src/components/editor/ImportResourceTemplate.jsx

```diff
@@ -85,20 +85,44 @@ class ImportResourceTemplate extends Component {
   updateStateFromServerResponse = (response, profileCount, updateKey) => {
     // HTTP status 409 == Conflict
     const showModal = response.status === 409 && this.state.createResourceError.length >= profileCount
-
-    const location = response.headers.location || ''
-    const newMessage = `${this.humanReadableStatus(response.status)} ${location}`
-    const newState = {
-      message: [...this.state.message, newMessage],
-      updateKey: updateKey
-    }
+    const newMessage = `${this.humanReadableStatus(response.status)} ${this.humanReadableLocation(response)}`
+    const newState = { updateKey: updateKey }
 
     if (showModal)
       newState.modalShow = true
 
+    // `response.ok` is a Boolean that returns true if the HTTP status is in the
+    // 200-299 range, indicating the HTTP request was successful
```

Comment:
superfluous comment?  Seems sorta SOP, and can be looked up somewhere?

Reply:
ok.